code-env/python/spec/requirements.txt

```diff
@@ -1,5 +1,5 @@
-google-api-python-client==1.12.3
-google-cloud-translate==2.0.1
+google-api-python-client==2.0.1
+google-cloud-translate==3.0.2
```

Comment:
Ah in fine, we are using the advanced API? :notbad:

Reply:
As you said v2 is a subset of v3 so I just changed the package to v3

Reply:
ok!